independent-publisher-connectors/SHRTCODE/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Chandra Sekhar Malla"
```

Comment:
@sekharmalla stackOwner property is missing. You can find more information in [this template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).